package.json

```diff
@@ -180,5 +180,6 @@
   },
   "ts-standard": {
     "project": "./tsconfig.json"
-  }
+  },
+  "version": "2.21.0"
```

Comment:
> Also, why don't we have a version entry in the `package.json`?

It is a legacy thing (from the time before v2) – NPM was not very useful back then.

@whizzzkid I think it will be easier for interop with release tools if you move the "source of truth" for version to `package.json`  and then make sure the build applies value from `package.json` to `add-on/manifest.common.json`.